plugin/trino-phoenix5/src/main/java/io/trino/plugin/phoenix5/PhoenixClient.java

```diff
@@ -205,6 +205,9 @@
     private static final String DATE_FORMAT = "y-MM-dd G";
     private static final DateTimeFormatter LOCAL_DATE_FORMATTER = DateTimeFormatter.ofPattern(DATE_FORMAT);
 
+    // Phoenix threshold for simplifying big IN predicates is 50k https://issues.apache.org/jira/browse/PHOENIX-6751
```

Comment:
It might be 50k, but will it evaluate in a performant way?

Reply:
I think Phoenix would set a lower limit if they knew it to perform badly.
Since this can also possibly depend on the hardware used by the source system, I don't think there is a universal answer to that question.
In any case, the limit I'm setting here is much lower.